src/NPlaylist/IPlaylistItem.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace NPlaylist
+{
+    public interface IPlaylistItem
+    {
+        IDictionary<string, string> Tags { get; }
+        // string location ???
```

Comment:
Add the path property
```suggestion
    string Path { get; }
```